src/main/java/nl/b3p/tailormap/api/persistence/OIDCConfiguration.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2023 B3Partners B.V.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+package nl.b3p.tailormap.api.persistence;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Version;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class OIDCConfiguration {
```

Comment:
This is named after a specific protocol. We may want to support other or new protocols in the future. We could name this more generic like `LoginConfiguration` and have subclasses for specific protocols?

Reply:
I think it's likely that OIDC will cover all needs for the coming time; if needed we can do a migration into an inheritance-mapped entity? And also, the only shared fields would be `name` (and `logo`, if added) - i'm uncertain that it's useful to use inheritance for that.